### DISCUSSION TITLE
不能升级的嘛？

### DISCUSSION THREAD

**Discussion by maxdeng007** [Moonvy/figmaEX#29] [Q&A]
我试了下原版的figma点一下检查更新，然后就升级了，但是EX的figma点检查更新，没有反应，然后在官网上重新下载了下，覆盖安装，发现还是 114.3版本，但是原版figma已经升级到了116.0.5，万能的群，这个是什么原因呢？

**Comment by yArna** [comment]
因为客户端更新后集成的功能就没有，所以禁用了更新。
每个客户端版本都得重新集成，所以客户端的一般的更新就不会跟随更新了，只有重要的客户端更新才会跟随。
Figma 的功能都是在网页中的，也就是说客户端即使是老版本也能用到最新的 Figma 功能，所以不用追求客户端最新版。
